### PR TITLE
Refresh node page

### PR DESCRIPTION
**Pull request by jrouzierinverse**
## Description

Refresh node page after deleting or updating node
## Impacts

Admin GUI
## Delete branch after merge

NO

### CODE REVIEW

**Comment by julsemaan** [comment]
This looks like it just refreshes the page without taking care of the search or the pagination...

**Comment by jrouzierinverse** [comment]
Please review

**Comment by julsemaan** [comment]
Missing news file entry

**Comment by julsemaan** [comment]
Tests completed and successful, search, pagination, sorting + advanced search works.

**Comment by jrouzierinverse** [comment]
Done

File: html/pfappserver/root/static/js/node.js [hunk 1]
```diff
@@ -421,6 +429,41 @@ NodeView.prototype.searchPagination = function(e) {
...
+            success: function(data) {
+                section.html(data);
+            },
+            errorSibling: status_container
+        });
+    });
+    return false;
+};
+
+
```
Comment (julsemaan) [inline html/pfappserver/root/static/js/node.js]: There are 3 lines separating both methods. Can you put it to 1 like the other methods.

Comment (jrouzierinverse) [inline html/pfappserver/root/static/js/node.js]: Done

**Pull request merged**